framework/config/src/main/java/org/apache/cloudstack/framework/config/ConfigKey.java

```diff
@@ -34,7 +34,7 @@
     public static final String CATEGORY_ALERT = "Alert";
 
     public static enum Scope {
-        Global, Zone, Cluster, StoragePool, Account, ManagementServer, ImageStore, Domain
+        Global, Zone, Cluster, StoragePool, Account, Domain, ManagementServer, ImageStore, Network
```

Comment:
Why change the `Domain` position on the enum?

Reply:
I thought it would be better if the Account and Domain sit together. :)

Reply:
@soreana, in some places in ACS, the enums' order affect the feature's behavior (for instance, the AccountType's ordinal is saved into the database). Would this reordering affect the feature's behavior or cause any type of regression?

Reply:
@JoaoJandre I didn't know that ACS uses the enums' order. It is surprising for me cause the order might change over time. :)
Anyway, I'm not aware of any regression but I'm redoing the changes just to be safe.

Reply:
@JoaoJandre @soreana `scope`is stored in the DB as text. I don´t think it matters. I could imagine using ordinal numbers in some enums but not for this one.

Reply:
Thanks for verifying that. I will resolve this conversation then :)